compliance_client/__init__.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from sys import exit,stderr
+from insights.client.connection import InsightsConnection
+from insights.client.config import InsightsConfig
+from insights.util.canonical_facts import get_canonical_facts
+from subprocess import Popen, PIPE, STDOUT
+from re import finditer
+from os import environ
+
+OSCAP_RESULTS_OUTPUT = '/tmp/oscap_results.xml'
+
+def get_config():
+    try:
+        return InsightsConfig(_print_errors=True).load_all()
+    except ValueError as e:
+        stderr.write('ERROR: ' + str(e) + '\n')
+        exit(1)
+
+# TODO: Not a typo! This endpoint gives OSCAP policies, not profiles
+# We need to update compliance-backend to fix this
+def get_policies(conn, hostname):
+    response = conn.session.get("https://ci.foo.redhat.com:1337/api/compliance/profiles", params={'hostname': hostname}, verify=False)
+    if response.status_code == 200:
+        return response.json()['data']
+    else:
+        return []
+
+def os_release():
+    grep = Popen("grep 7\.[0-9]* /etc/redhat-release", shell=True, stdout=PIPE, stderr=STDOUT).stdout.read()
```

Comment:
Using shell=True can be a security hazard https://docs.python.org/2/library/subprocess.html#frequently-used-arguments. And instead of parsing `/etc/redhat-release` use `platform` module:
```
from platform import linux_distribution

linux_distribution()
```